ipn/ipnlocal/local.go

```diff
@@ -273,7 +272,10 @@ type LocalBackend struct {
 	serveConfig         ipn.ServeConfigView // or !Valid if none
 	activeWatchSessions set.Set[string]     // of WatchIPN SessionID
 
-	serveListeners     map[netip.AddrPort]*serveListener // addrPort => serveListener
+	webClient          webClient
```

Comment:
this is just movement, but can this be nil?

Reply:
no.  It's just a container for two related pointers.  We do nil checks on the individual pointer values